lib/ember-cli/deploy/redis.rb

```diff
@@ -0,0 +1,82 @@
+require "active_support/core_ext/object/blank"
+require "ember-cli/deploy/page"
+require "redis"
+
+module EmberCLI
+  module Deploy
+    class Redis
+      def initialize(namespace:, index_html: nil)
```

Comment:
@aaronrenner could you share an example where you're sharing Redis connections?

Where are you creating them? 

Where are you closing / destroying them?

How are you pooling the connections?